includes/EditPage.php

```diff
@@ -2556,7 +2556,7 @@ protected function showStandardInputs( &$tabindex = 2 ) {
 	 */
 	protected function showConflict() {
 		global $wgOut;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)